.env.test

```diff
@@ -0,0 +1,16 @@
+DEBUG=true
+SECRET_KEY='tests'
+DATABASE_URL='postgresql:///opencraft'
```

Comment:
@mtyaka Django already automatically does this, cf https://docs.djangoproject.com/en/1.8/topics/testing/overview/#the-test-database : 

> By default the test databases get their names by prepending test_ to the value of the NAME settings for the databases defined in DATABASES.

But it doesn't hurt to use a different name here, so I'm changing the DB name to `opencraft-test` here.